runtime/karura/src/lib.rs

```diff
@@ -1060,6 +1062,15 @@ impl OnUnbalanced<NegativeImbalance> for DealWithFees {
 		}
 	}
 }
+impl TakeRevenue for DealWithFees {
```

Comment:
I would prefer not to reuse the `DealWithFees`. We should make each struct type concern their own business. Declare a new struct before `XcmConfig` and name it `ToTreasury`.